app.js

```diff
@@ -0,0 +1,81 @@
+const humburger = document.querySelector('.humburger');
+const mobMenu = document.querySelector('.desk_top');
+const li2 = document.querySelector('.nem2');
+const li3 = document.querySelector('.nem3');
+const li4 = document.querySelector('.nem4');
+const li5 = document.querySelector('.nem5');
+
+li2.addEventListener('click', () => {
+  humburger.classList.remove('activ');
+});
+li3.addEventListener('click', () => {
+  humburger.classList.remove('activ');
+  mobMenu.classList.toggle('activ');
+});
+li4.addEventListener('click', () => {
+  humburger.classList.remove('activ');
+  mobMenu.classList.toggle('activ');
+});
+li5.addEventListener('click', () => {
+  humburger.classList.remove('activ');
+  mobMenu.classList.toggle('activ');
+});
+
+humburger.addEventListener('click', () => {
+  humburger.classList.toggle('activ');
+  mobMenu.classList.toggle('activ');
+});
+
+const speaker1 = {
+  imageSrc: './assets/barack-obama.jpeg',
+  name: 'Ghislain Mitahi',
+  rank: 'Cheif Executive Officer (CEO)',
+  id: 'firstSPeaker',
+  text: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's",
+};
+
+const speaker2 = {
+  imageSrc: './assets/barack-obama.jpeg',
+  name: 'ZUZU zazu',
+  rank: 'Cheif Marketing',
+  id: 'secondarySPeaker',
+  text: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's",
+};
+
+const speaker3 = {
+  imageSrc: './assets/barack-obama.jpeg',
+  name: 'Blaise Blaise',
+  rank: 'IT',
+  id: 'thirdSPeaker',
+  text: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's",
+};
+
+const speaker4 = {
+  imageSrc: './assets/barack-obama.jpeg',
+  name: 'TRESOR TRESOR',
+  rank: 'GRENERAL DIRECTOR',
+  id: 'fourthSPeaker',
+  text: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's",
+};
```

Comment:
- [ ]  It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description. you have only 4 kindly go through the requirement
### Your design
![his design](https://user-images.githubusercontent.com/81727988/181704163-f2d5744f-f741-4ce8-a9e6-d34875c6724a.png)
### The design provided by microverse
![figma](https://user-images.githubusercontent.com/81727988/181704247-1507f04e-1e39-48c2-9241-e55dc1f61fc0.png)